pkgs/development/python-modules/recipe-scrapers/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, beautifulsoup4
+, extruct
+, language-tags
+, regex
+, requests
+, pytestCheckHook
+, pytest
+, responses
+}:
+
+buildPythonPackage rec {
+  pname = "recipe-scrapers";
+  version = "14.14.0";
+
+  src = fetchFromGitHub {
+    owner = "hhursev";
+    repo = "recipe-scrapers";
+    rev = version;
+    sha256 = "sha256-3qrjNd1jX4JP3qG9YX8MQqwPh8cvfkZa1tEk0uCwego=";
+  };
+
+  propagatedBuildInputs = [
+    beautifulsoup4
+    extruct
+    language-tags
+    regex
+    requests
+  ];
+
+  doCheck = true;
+
+  checkInputs = [
+    pytestCheckHook
+    pytest
+    responses
+  ];
+
+  disabledTestPaths = [
+    # Import errors in this file
```

Comment:
```
templates/test_scraper.py:1: in <module>
    from recipe_scrapers.template import Template
E   ModuleNotFoundError: No module named 'recipe_scrapers.template'
```

IIUC there is some `PYTHONPATH` shenanigans happening in upstream's tests, since what it calls `recipe_scrapers` looks to be the file at `templates/scraper.py` ([here](https://github.com/hhursev/recipe-scrapers/blob/3a3d2fdde8c5d3eedeb62894f509f2f69ca4ab9c/templates/scraper.py)).

Reply:
Aaaah no, the `templates` folder is just some starter template for when they want to write a new scraper.

It's not intended to be working code.

Will update the comment.